src/test/kotlin/com/tschuchort/compiletesting/KotlinCompilationTests.kt

```diff
@@ -623,6 +624,17 @@ class KotlinCompilationTests {
 		assertClassLoadable(result, "${KotlinTestProcessor.GENERATED_PACKAGE}.${KotlinTestProcessor.GENERATED_JAVA_CLASS_NAME}")
 	}
 
+	@Test
+	fun `checks configuration with pluginClasspaths`() {
+		val result = defaultCompilerConfig().apply {
+			sources = listOf(SourceFile.kotlin("kSource.kt", "class KSource"))
+			pluginClasspaths = listOf(File("./plugin.jar"))
```

Comment:
I was about to ask where this `./plugin.jar` came from before I saw that you are testing failure.

Reply:
Good point, thanks! I gonna change the name to avoid P.O.L.A.

Reply:
Fixed!

Reply:
What's P.O.L.A. by the way?

Reply:
Sorry, I didn't put the link: **[P.O.L.A.](http://wiki.c2.com/?PrincipleOfLeastAstonishment)** 
I astonished you with `./plugin.jar`. 
Now it's `./non-existing-plugin.jar`. It communicates better, I think, and you wouldn't wonder where it comes from.